test/test-utils.js

```diff
@@ -331,3 +337,123 @@ export const adminCreateCategory =
     }
     return res;
   };
+export const createVoucherType = (chain) => async (description, duration) => {
+  const VOUCHER_HUB_ABI = {
+    inputs: [
+      {
+        internalType: 'string',
+        name: 'description',
+        type: 'string',
+      },
+      {
+        internalType: 'uint256',
+        name: 'duration',
+        type: 'uint256',
+      },
+    ],
+    name: 'createVoucherType',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  };
+  const voucherHubContract = new Contract(
+    chain.voucherHubAddress,
+    VOUCHER_HUB_ABI,
+    chain.voucherManagerWallet,
+  );
+  return voucherHubContract.createVoucherType(description, duration);
```

Comment:
this returns a `Promise<Transaction>` we should wait for the tx validation, inspect the log, and find the created `VoucherTypeId` to return it.